main/main.zkasm

```diff
@@ -108,7 +108,7 @@ endCheckRLP:
 txLoop:
         $ => A          :MLOAD(pendingTxs)
         A-1 => A        :MSTORE(pendingTxs)
-        A               :JMPN(processTxsEnd)
+        A               :JMPN(processTxsEnd) ; TODO CHECK JUMPN only checks 64 bits
```

Comment:
`pendingTxs` is computed internally.
This check will give us a maximum number of 2³² transactions to process in one batch which I think it is save xD